lib/classes/PluginManager.js

```diff
@@ -148,7 +150,23 @@ class PluginManager {
     if (pluginsObject.localPath) {
       module.paths.unshift(pluginsObject.localPath);
     }
-    this.loadPlugins(pluginsObject.modules);
+    this.loadPlugins(pluginsObject.modules
+      .filter(name => name !== '@serverless/enterprise-plugin'));
+  }
+
+  loadEnterprisePlugin() {
+    module.paths.unshift(path.join(__dirname, '../../node_modules'));
+    this.loadPlugins(['@serverless/enterprise-plugin']);
+    // eslint-disable-next-line global-require
```

Comment:
We just got update to eslint config where this [rule is turned off](https://github.com/serverless/serverless/blob/32a9b9954ba9d06d46fa71595cb3e68beb04f5c5/.eslintrc.js#L7). So this line can be removed now